tools/pdffontsinfo.py

```diff
@@ -0,0 +1,222 @@
+    elif args.rotation:
+        rotation = int(args.rotation)
+    elif args.pagenos:
+        pagenos.update(int(x)-1 for x in args.pagenos.split(','))
+    elif args.maxpages:
+        maxpages = int(args.maxpages)
+    elif args.disable_caching:
+        caching = False
+    elif args.no_layout:
+        laparams = None
+    elif args.all_texts:
+        laparams.all_texts = True
+    elif args.detect_vertical:
+        laparams.detect_vertical = True
+    elif args.char_margin:
+        laparams.char_margin = float(args.char_margin)
+    elif args.word_margin:
+        laparams.word_margin = float(args.word_margin)
+    elif args.line_margin:
+        laparams.line_margin = float(args.line_margin)
+    elif args.boxes_flow:
+        laparams.boxes_flow = float(args.boxes_flow)
+
+    #
+    PDFDocument.debug = debug
+    PDFParser.debug = debug
+    CMapDB.debug = debug
+    PDFPageInterpreter.debug = debug
+    #
+
+    rsrcmgr = PDFResourceManager(caching=caching)
+
+    if outfile:
+        outfp = open(outfile, 'w')
+    else:
+        outfp = sys.stdout
+
+    device = PDFConverter(
+        rsrcmgr, outfp, laparams=laparams)
+
+    for fname in args.input:
+        with open(fname, 'rb') as fp:
+            interpreter = PDFPageInterpreter(rsrcmgr, device)
+            for page in PDFPage.get_pages(
+                fp,
+                pagenos,
+                maxpages=maxpages,
+                password=password,
+                caching=caching,
+                check_extractable=True,
+            ):
+                page.rotate = (page.rotate + rotation) % 360
+                interpreter.process_page(page)
+
+        l_font_names = []
+        l_font_types = []
+        l_font_enc = []
+        l_font_uni = []
+
+        # Collecting data from sources
+        for fontkey in rsrcmgr._cached_fonts:
+            # Font = class (PDFType1Font etc)
+            font = rsrcmgr._cached_fonts[fontkey]
+            # Type = TrueType/Type 1/Type 3/Type CID
+            font_type = font.__class__.__name__
+            # Name = Helvetica etc
+            font_name = font.basefont
+
+            font_enc = font.get_encoding_name()
+
+            font_uni = font.get_toUnicode()
+
+            l_font_types.append(font_type)
+            l_font_names.append(font_name)
+            l_font_enc.append(font_enc)
+            l_font_uni.append(font_uni)
+
+        fonts = {}
+        fonts['names'] = l_font_names
+        fonts['types'] = l_font_types
+        fonts['enc'] = l_font_enc
+        fonts['uni'] = l_font_uni
+
+        device.close()
+        outfp.write(fname+" fonts:\n")
+        outfp.write(fonts2txt(fonts))
+        outfp.write('\n')
+
+        # delete the cache
+        rsrcmgr._cached_fonts = {}
+
+    if outfile:
+        outfp.close()
+
+
+def fonts2txt(fonts):
+    ''' Transforms the font information in a nice text table following
+        this pattern:
+
+        Name                       Type     Encoding             Uni
```

Comment:
I think changing `Uni` to `Unicode` would make the table more understandable for the user when they run the command

Reply:
Fixed! Thanks for the suggestion.